applications/matter_bridge/README.rst

```diff
@@ -47,6 +47,35 @@ The application supports bridging the following Matter device types:
 * Temperature Sensor
 * Humidity Sensor
 
+The application supports over-the-air (OTA) device firmware upgrade (DFU) using one of the two following protocols:
+
+* Matter OTA update protocol that uses the Matter operational network for querying and downloading a new firmware image.
```

Comment:
```suggestion
* Matter OTA update protocol.
  Querying and downloading a new firmware image is done using the Matter operational network.
```
Wanted to break this into two sentences so the protocol is its own sentence, which meant rearranging the rest of it. Make sure that the meaning is still the same.